test/minimal_init.lua

```diff
@@ -0,0 +1,73 @@
+local execute = vim.api.nvim_command
+local fn = vim.fn
+
+local install_path = "/tmp/site/pack/packer/opt/packer.nvim"
+
+if fn.empty(fn.glob(install_path)) > 0 then
+  execute("!git clone https://github.com/wbthomason/packer.nvim " .. install_path)
+  execute "packadd packer.nvim"
```

Comment:
you can remove this because we already have one on line 11

Reply:
copypasta error